pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2881,7 +2881,7 @@ func (c *Cloud) ensureSecurityGroup(name string, description string) (string, er
 		return "", fmt.Errorf("created security group, but id was not returned: %s", name)
 	}
 
-	err := c.tagging.createTags(c.ec2, groupID, ResourceLifecycleOwned, nil)
+	err := c.tagging.createTags(c.ec2, groupID, ResourceLifecycleOwned, getLoadBalancerAdditionalTags(annotations))
```

Comment:
Instead of assuming that this is a ELB security group, how about if we pass in `getLoadBalancerAdditionalTags(annotations)` to ensureSecurityGroup.  I suspect we don't call the function anywhere else, but it makes the method less surprising.